gateway-ha/src/test/java/io/trino/gateway/TrinoGatewayRunner.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.gateway;
+
+import io.trino.gateway.ha.HaGatewayLauncher;
+import org.testcontainers.containers.MySQLContainer;
+import org.testcontainers.containers.PostgreSQLContainer;
+import org.testcontainers.containers.TrinoContainer;
+
+import java.util.List;
+
+import static org.testcontainers.utility.MountableFile.forClasspathResource;
+
+public final class TrinoGatewayRunner
+{
+    private TrinoGatewayRunner() {}
+
+    public static void main(String[] args)
+            throws Exception
+    {
+        TrinoContainer trino1 = new TrinoContainer("trinodb/trino");
+        trino1.setPortBindings(List.of("8081:8080"));
+        trino1.start();
+        TrinoContainer trino2 = new TrinoContainer("trinodb/trino");
+        trino2.setPortBindings(List.of("8082:8080"));
+        trino2.start();
+
+        PostgreSQLContainer<?> postgres = new PostgreSQLContainer("postgres");
```

Comment:
Should we fix version here as well?

Reply:
I agree. Am not too familiar with postgres versions, but mysql versions DO have issues (like field size or primary key index issue) so it might not be compatible with old versions when developing.